src/queryMetrics/clientSideMetrics.ts

```diff
@@ -0,0 +1,58 @@
+import { QueryMetricsUtils } from "./queryMetricsUtils";
+
+export class ClientSideMetrics {
+    private requestCharge: number;
+    constructor(requestCharge: number) {
+        // Constructor
+        if (!QueryMetricsUtils.isNumeric(requestCharge)) {
+            throw new Error("requestCharge is not a numeric type");
+        }
+
+        this.requestCharge = requestCharge;
+    }
+
+    /**
+     * Gets the RequestCharge
+     * @memberof ClientSideMetrics
+     * @instance
+     * @ignore
+     */
+    public getRequestCharge() {
```

Comment:
Can we use the getters / setters that supported by typescript?

Reply:
In this case, if we make it readonly, we can just make it public, right?